pjsip/include/pjsua2/call.hpp

```diff
@@ -333,6 +333,13 @@ struct CallSetting
      */
     MediaDirVector mediaDir;
 
+    /**
+     * User defined Call-ID to be sent out with outgoing INVITE
+     *
+     * This setting will only be used when creating a new outgoing call via Call::makeCall().
```

Comment:
Similar as above in `pjsua_call_setting`:
- Info/warning about no validity check.
- Col width > 80?